app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="center jumbotron">
+    <h1>Welcome to PokeLearning</h1>
+
+    <h2>This is the home page!</h2>
+
+    <%= link_to "Sign up now!", signup_path, class: "btn btn-lg btn-primary" %>
```

Comment:
@shaniajubelag This causes error because `signup_path` does not exist in your routes yet.
Change this  to `#` for temporary value